dotcom-rendering/scripts/webpack/server.js

```diff
@@ -18,6 +18,12 @@ module.exports = () => ({
 		require('webpack-node-externals')({
 			allowlist: [/^@guardian/],
 		}),
+		({ request }, callback) => {
+			return process.env.NODE_ENV === 'development' &&
```

Comment:
Would it be worth leaving a comment here saying it's in regard to speeding up the development build. I know the check indicates this, but it could be nice to say why.